src/lambda/models/samLambdaRuntime.ts

```diff
@@ -127,9 +138,12 @@ function getRuntimesForFamily(family: RuntimeFamily): Set<Runtime> | undefined {
 export function createRuntimeQuickPick(params: {
     buttons?: vscode.QuickInputButton[]
     currRuntime?: Runtime
+    filteredRuntimes?: ImmutableSet<Runtime>
     runtimeFamily?: RuntimeFamily
 }): vscode.QuickPick<vscode.QuickPickItem> {
-    const runtimes = params.runtimeFamily
+    const runtimes = params.filteredRuntimes
```

Comment:
This nested conditional operator will be susceptible to future maintenance mistakes. 
Its your call whether to leave it or make changes.

Reply:
?